source/Handlebars/Compiler/Translation/Expression/UnencodedStatementVisitor.cs

```diff
@@ -20,10 +20,14 @@ protected override Expression VisitStatementExpression(StatementExpression sex)
                 var writer = CompilationContext.Args.EncodedWriter;
                 var suppressEncoding = writer.Property(o => o.SuppressEncoding);
                 
+                var existingValue = new ExpressionContainer<bool>(Expression.Variable(typeof(bool)));
+                
                 return Block()
+                    .Parameter(existingValue)
```

Comment:
Can be improved by introducing variable inline:
```suggestion
                return Block()
                    .Parameter<bool>(out var existingValue)
```